cmd/nomos/version/version.go

```diff
@@ -257,7 +256,7 @@ type entry struct {
 func entries(vs map[string]vErr) []entry {
 	currentContext, err := restconfig.CurrentContextName()
 	if err != nil {
-		fmt.Printf("Failed to get current context name with err: %v\n", errors.Cause(err))
+		fmt.Printf("Failed to get current context name with err: %v\n", err)
```

Comment:
Good question.  I did think about this one for a bit.  It's not clear to me why we would want to ignore the error at the top of the stack in case these are wrapped.  The danger of calling the new equivalent of the Cause() is that you might just get a `nil` https://pkg.go.dev/errors#Unwrap.

In this case, the original author didn't leave any trace of why they chose to ignore the wrapping error.  In most circumstances I would want to see the full error.